DATA/EQUIPMENT/weapon_equip.ini

```diff
@@ -49680,3 +49680,77 @@ auto_turret = false
 turn_rate = 90
 LODranges = 0, 100, 200, 500, 1000
 lootable = true
+
+;; Small Repairtool
+
+[Explosion]
+nickname = gun_codename_repair_special01_explosion
+effect = xsx_repairflash01
+lifetime = 0, 0
+process = disappear
+strength = 100
+radius = 1
+impulse = 0
+
+[Munition]
+nickname = gun_codename_repair_special01_ammo
+hp_type = hp_gun
+requires_ammo = false
+explosion_arch = gun_codename_repair_special01_explosion
+hit_pts = 2
+hull_damage = -2000
+energy_damage = 0
+weapon_type = W_Laser
+one_shot_sound = repair_gun_light02
+munition_hit_effect = li_laser_01_impact

```

Comment:
Hit effect additional to the explosion? Is that intended?

Reply:
Yes, because the Explosion effect build up and to have smoother transition it will have the direct hit effect from the Munition.